Makefile

```diff
@@ -113,14 +113,14 @@ HELM_HOOKS = preinstall postinstall predelete
 $(HELM_HOOKS): $(GO_FILES)
 	GOARCH=$(GOARCH) GOOS=linux CGO_ENABLED=0 go build -ldflags "$(GO_LDFLAGS)" -o out/$@ $(REPOPATH)/helm-hooks/$@
 
-.PHONY: %-image
```

Comment:
I'm missing something here: what's the reason for this diff? It appears to re-order this with %-test-image, but I'm not sure of the underlying reason behind it.

Reply:
Hi @aysylu 
I got the following error when I run `make {preinstall,postinstall,predelete}-test-image`:
```
$ make preinstall-test-image
GOARCH=amd64 GOOS=linux CGO_ENABLED=0 go build -ldflags "-extldflags "-static" -X github.com/grafeas/kritis/cmd/kritis/version.Commit=7dec1d24e9c658f8c8266fd9e6fedc679ce6df88 -X github.com/grafeas/kritis/cmd/kritis/version.Version=v0.1.0 -w -s " -o out/preinstall github.com/grafeas/kritis/helm-hooks/preinstall
GOARCH=amd64 GOOS=linux CGO_ENABLED=0 go build -ldflags "-extldflags "-static" -X github.com/grafeas/kritis/cmd/kritis/version.Commit=7dec1d24e9c658f8c8266fd9e6fedc679ce6df88 -X github.com/grafeas/kritis/cmd/kritis/version.Version=v0.1.0 -w -s " -o out/postinstall github.com/grafeas/kritis/helm-hooks/postinstall
GOARCH=amd64 GOOS=linux CGO_ENABLED=0 go build -ldflags "-extldflags "-static" -X github.com/grafeas/kritis/cmd/kritis/version.Commit=7dec1d24e9c658f8c8266fd9e6fedc679ce6df88 -X github.com/grafeas/kritis/cmd/kritis/version.Version=v0.1.0 -w -s " -o out/predelete github.com/grafeas/kritis/helm-hooks/predelete
docker build -t gcr.io/kritis-project/preinstall-test:7dec1d24e9c658f8c8266fd9e6fedc679ce6df88 -f helm-hooks/Dockerfile . --build-arg stage=preinstall-test
Sending build context to Docker daemon  59.06MB
Step 1/11 : FROM golang:1.10
 ---> 9198b7a162bb
Step 2/11 : WORKDIR /go/src/github.com/grafeas/kritis
 ---> Using cache
 ---> ab6410231871
Step 3/11 : COPY . .
 ---> a7218b1a724a
Step 4/11 : ARG stage
 ---> Running in 016b08cbfac3
Removing intermediate container 016b08cbfac3
 ---> 4a367c6207d9
Step 5/11 : RUN make ${stage}
 ---> Running in 2769766eede6
make: *** No rule to make target 'preinstall-test'.  Stop.
The command '/bin/sh -c make ${stage}' returned a non-zero code: 2
make: *** [preinstall-test-image] Error 2
```

As you can see `No rule to make target 'preinstall-test'`, the `%-image` was matched unexpectedly.

If I re-order `%-image` with `%-test-image`, it works properly.
That's why I made this change.

Reply:
Thanks for clarifying!